.github/workflows/graph-pixi-tests.yml

```diff
@@ -0,0 +1,143 @@
+name: Graph Pixi Tests
+
+on:
+  push:
+    paths:
+      - 'cypress/e2e/pixi-interaction/**'
+      - '.github/workflows/graph-pixi-tests.yml'
+
+jobs:
+  lint:
```

Comment:
I'm not sure if I missed this before or you added it recently, but I don't think `lint` should be run in this worlkflow. It is redundant with the lint that is run during main `v3.yml` workflow.